neuroquery/datasets.py

```diff
@@ -72,12 +73,19 @@ def fetch_peak_coordinates(data_dir=None):
         return str(out_file)
     print("Downloading coordinates")
     resp = requests.get(
-        "https://raw.githubusercontent.com/neuroquery/neuroquery_data/"
-        "master/training_data/coordinates.csv"
+        "https://raw.githubusercontent.com/neuroquery/neuroquery_data/master/data/data-neuroquery_version-1_coordinates.tsv.gz"
```

Comment:
could we split this line so it fits in 80 columns? between parentheses you can split a literal strings on several lines like this:
```python
    resp = requests.get(
        "https://raw.githubusercontent.com/neuroquery/neuroquery_data/"
        "master/data/data-neuroquery_version-1_coordinates.tsv.gz"
    )
```
and it is exactly equivalent to putting the whole string on one line